### PR TITLE
metrics for addtional mounted harddisks

### PR DESCRIPTION
**Pull request by jaswanth098**

### CODE REVIEW

File: serverstats/serverstats.py [hunk 1]
```diff
@@ -102,10 +107,24 @@ def get_system_metrics():
...
+
         #network traffic
         network_traffic=network_traffic,
     )
 
+    if mount_points:
+        stats["mount_points"] = [
+            dict(
+                total=float(disk.total),
+                usage=float(disk.used),
```
Comment (jaswanth098) [inline serverstats/serverstats.py]: Need to change disk variable  here

File: serverstats/serverstats.py [hunk 2]
```diff
@@ -102,10 +107,24 @@ def get_system_metrics():
...
+                free=float(disk.free),
+                usage_percent=float(disk.percent),
+                free_percent=float((disk.free / disk.total * 100)),
+                mount_point=i
+            )
+            for i in mount_points],
```
Comment (jaswanth098) [inline serverstats/serverstats.py]: Need to change variable i to meaningfully name

**Comment by supriyopaul** [comment]
I created a new docker image tagged as deepcompute/serverstats:0.2.1 from this code. If it works ok for Anil then we can merge the code . @jaswanth098

**Comment by jaswanth098** [comment]
@supriyopaul I have changed the code a little bit more could you please rebuild docker and push it

**Pull request closed without merging**